steps/discover.sh

```diff
@@ -69,6 +69,18 @@ elif [ -z "${REPOS}" ] || [ ! -e "${REPOS}" ]; then
   samples-filter filter "${fargs[@]}"
   rm "${csv}"
   mv "${nosamples}" "${csv}"
+
+  maintained=${TARGET}/maintained.csv
+    declare -a margs=( \
+      "--repositories=${csv}" \
+      "--output=${maintained}" \
+      "--key=${GIGACHAT_KEY}" \
+      "--model=${GIGACHAT_MODEL}"
```

Comment:
@dzhovi can we ask `repo-reasoner` to save its logs somewhere? It will be very interesting to see why some repositories were filtered out.